modules/meeting/app/components/meetings/sidebar/details_component.html.erb

```diff
@@ -85,6 +85,27 @@
               end
             end
           end
+          details.with_row(mt: 2, classes: 'op-meeting-sidebar-details-participants') do
+            flex_layout do |flex|
+              render(Primer::Beta::Octicon.new(icon: 'people'))
+              flex.with_row(classes: 'd-none', data: { 'meetings-sidebar-details-target': "hiddenParticipants" }) do
```

Comment:
Please do not set the primer classes by yourself, but use the `system_arguments` for that.